website/docs/extensions/install/index.md

```diff
@@ -8,24 +8,28 @@ keywords: [podman desktop, extension, publishing]
 
 # Installing a Podman Desktop extension
 
-Consider installing Podman Desktop extensions to enrich the default capabilities of Podman Desktop.
+Installing an extension is a great way to expand the capability of Podman Desktop.
 
 #### Prerequisites
 
-- The extension Open Container Initiative (OCI) image name, such as: `quay.io/fbenoit/my-first-extension`.
-
-  See [Publishing a Podman Desktop extension](/docs/extensions/publish).
+- Extensions are packaged as container images. The extension will either direct you to the catalog tab, or provide you with a link to the image (ex. `quay.io/example/extension-name`).
```

Comment:
'Extensions are packaged as container images.' feels irrelevant/too much detail here. Maybe could be in the custom section, like 'extensions can be installed from their container image'.

'The extension will ...' - it feels like it should be worded by starting with "There are two ways to install an extension, ..."